### DISCUSSION TITLE
發問盡量以Python內容為主

### DISCUSSION THREAD

**Discussion by KozakHou** [1101-GS4719-C/Python-Community-GS4719-C#1401] [General]
身為學生，我不能同意的更多。

為什麼要以Python內容為主? 你們可能會問 [ 為 什 麼 ! ]

恰好我現在有空，就來告訴各位，為什麼。

首先，各位，我們在上Python的課，這也是為什麼，討論內容理應要以Python為主。

巴爾扎克曾講過，不久他開始散佈出老年人的那點兒涼意，像北風一般把人世間家用的感情都吹涼了，尤其他是個又窮又醜的老人，那不是老上加老嗎？這是人生到了冬季，鼻子通紅，腮幫灰白，手腳麻木的冬季！ 這段話讓我所有的疑惑頓時豁然開朗。看看別人，再想想自己，會發現問題的核心其實就在你身旁。帶著這些問題，我們一起來審視為何要學Python

第一，語法簡單、第二，適用性廣泛、第三，操作簡單(好像跟第一差不多)、第四，超多人用，你不怕問題沒人遇過。
所以，為何要學Python ? 你們捫心自問， 為什麼不 !

**Comment by sumairu0727** [comment]
我知道這是python的課，但我記得當初老師會希望我們在這邊討論的主因是希望我們熟悉github的使用，對於教學通常有問題在課堂上都有助教跟老師可以問。
當然可能在學習本語言會有問題需要在此提問，但要賺到70點的分數勢必不可能以python的討論為主。況且我們不是所有人都是資電走向，不可能每個人都花那麼多時間學習python並發現問題。望周知。

**Reply by chia-kai-chang** [reply]
也是可以討論科技相關的新知或是新聞

**Comment by Brownnnn78** [comment]
我覺得要我們在一個科目裡面去尋找問題勢必要我們在學習該科目時才會碰到問題，當我們這些在python課程以外完全沒機會碰到他的人要憑空想像一個問題實在是很困難呀。
當然我也知道在生活中會寫一些小程式幫忙自己處理簡單且重複的問題是很方便的，但比起這時才開始研究python怎麼撰寫，不如自己操作，畢竟除了這些課程以外，不會有作業讓我們處理大量資訊。
